src/Extensions/Hosting.Services.Web/HostBuilderExtensions.cs

```diff
@@ -22,12 +23,14 @@ public static class HostBuilderExtensions
 			this ServiceFabricHostBuilder<OmexStatelessService, StatelessServiceContext> builder,
 			string name,
 			ServiceFabricIntegrationOptions options,
-			Action<IWebHostBuilder>? builderExtension = null)
+			Action<IWebHostBuilder>? builderExtension = null,
+			Action<KestrelServerOptions>? kestrelOptions = null)
```

Comment:
Probably, it's better to use the second overload with both parameters `Action<WebHostBuilderContext, KestrelServerOptions>` since it would provide more configurability and user won't need to specify the type in Activity so it won't be a lot of overhead.

Reply:
Sounds good, changed Action<WebHostBuilderContext> to Action<WebHostBuilderContext, KestrelServerOptions>